src/App.js

```diff
@@ -0,0 +1,23 @@
+import ReactDOM from "react-dom";
+import React from "react";
+import {Widget} from "./widget/widget";
+
+
+const e = React.createElement;
+
+export class App extends React.Component {
+    constructor(props) {
```

Comment:
You might get a warning about a useless constructor here? if you are not getting a warning then ignore this comment

Reply:
I have no warning on IntelliJ